stdlib/LinearAlgebra/src/dense.jl

```diff
@@ -708,25 +708,32 @@ function exp!(A::StridedMatrix{T}) where T<:BlasFloat
         # Compute U and V: Even/odd terms in Padé numerator & denom
         # Expansion of k=1 in for loop
         P = A2
-        U = mul!(C[4]*P, true, C[2]*I, true, true) #U = C[2]*I + C[4]*P
-        V = mul!(C[3]*P, true, C[1]*I, true, true) #V = C[1]*I + C[3]*P
```

Comment:
Was the broadcasting "hidden" here?

Reply:
Yes, `C[3]*P` is evaluated as `C[3] .* P`, which goes through the whole broadcast machinery.